thsvs_simpler_vulkan_synchronization.h

```diff
@@ -891,16 +891,16 @@ void thsvsGetVulkanImageMemoryBarrier(
                 break;
         }
 
+        pVkBarrier->newLayout = layout;
 #ifdef THSVS_ERROR_CHECK_MIXED_IMAGE_LAYOUT
         assert(pVkBarrier->newLayout == VK_IMAGE_LAYOUT_UNDEFINED ||
                pVkBarrier->newLayout == layout);
 #endif
-        pVkBarrier->newLayout = layout;

```

Comment:
Same comment as above - newLayout should just be initialised to VK_IMAGE_LAYOUT_UNDEFINED.